### PR TITLE
Fix/folders

### PR DESCRIPTION
**Pull request by Michael-Reefe**

### CODE REVIEW

**Comment by Michael-Reefe** [comment]
Fixes a minor issue where multiple tickets run at once would save to the same folder

File: main/drivers/driver.py [hunk 1]
```diff
@@ -142,3 +155,21 @@ def start_time(ticket_object):
...
+        Alphanumeric list to be sorted.
+
+    Returns
+    -------
+    LIST
+        Sorted list
+    """
+    conversion = lambda item: int(item) if item.isdigit() else item
+    alphanum_sorting = lambda key: [conversion(n) for n in re.split('([0-9]+)', key)]
+    return sorted(_list, key=alphanum_sorting)
```
Comment (Kakon24) [inline main/drivers/driver.py:175]: This is great stuff.

File: main/drivers/driver.py [hunk 2]
```diff
@@ -75,24 +76,36 @@ def run(obs_tickets, data=None, config=None, _filter=None, logger=None, shutdown
...
+        occurrences = folder.count(folder[i])
+        if occurrences < 2 and not os.path.exists(folder[i]):
+            os.makedirs(folder[i])
+        else:       # If occurrences >= 2 or os.path.exists(folder[i])
+            for letter in string.ascii_letters:
+                if not os.path.exists(folder[i] + letter):
+                    folder[i] += letter
+                    os.makedirs(folder[i])
+                    break
+    print('New directories for tonight\'s observing have been made!')
```
Comment (Kakon24) [inline main/drivers/driver.py]: This should be a log?

File: main/observing/observation_run.py [hunk 3]
```diff
@@ -98,51 +98,36 @@ def everything_ok(self):
...
-                            self.guider.onThread(self.guider.guiding_procedure)
-                    elif self.current_ticket != self.observation_request_list[-1]:
-                        self._startup_procedure()
-                else:
-                    print('Weather is still too poor to resume observing.')
-                    self.everything_ok()
             else:
-                time.sleep(60*self.config_dict.min_reopen_time)
                 while self.conditions.weather_alert.isSet():
+                    print("Still waiting for good conditions to reopen.")
```
Comment (Kakon24) [inline main/observing/observation_run.py]: Should be logged?

File: main/observing/observation_run.py [hunk 4]
```diff
@@ -210,6 +195,17 @@ def observe(self):
         -------
         None.
         """
+        tz_0 = self.observation_request_list[0].start_time.tzinfo
+        current_time = datetime.datetime.now(tz_0)
+        if self.observation_request_list[0].start_time > current_time:
+            print("It is not the start time {} of {} observation, "
```
Comment (Kakon24) [inline main/observing/observation_run.py]: Logger

**Review by Kakon24 — CHANGES_REQUESTED** [review]
Minor changes

**Review by Kakon24 — APPROVED** [review]

**Pull request merged**